compose/src/main/kotlin/com/yourssu/handy/compose/icons/HandyIcons.kt

```diff
@@ -0,0 +1,50 @@
+package com.yourssu.handy.compose.icons
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.StrokeJoin
+import androidx.compose.ui.graphics.vector.DefaultFillType
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.PathBuilder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+
+object HandyIcons {
+    object Filled
+    object Line
+}
+
+inline fun handyIcon(
+    name: String = "",
+    block: ImageVector.Builder.() -> Unit
+): ImageVector = ImageVector.Builder(
+    name = name,
+    defaultWidth = HandyIconDimension.dp,
+    defaultHeight = HandyIconDimension.dp,
+    viewportWidth = HandyIconDimension,
+    viewportHeight = HandyIconDimension
+).apply(block).build()
+
+inline fun ImageVector.Builder.handyPath(
+    fillAlpha: Float = 1.0f,
+    strokeAlpha: Float = 1.0f,
+    strokeLineWidth: Float = 1.0f,
+    pathFillType: PathFillType = DefaultFillType,
+    pathBuilder: PathBuilder.() -> Unit
+) = path(
+    fill = SolidColor(Color.Black),
+    fillAlpha = fillAlpha,
+    stroke = null,
+    strokeAlpha = strokeAlpha,
+    strokeLineWidth = strokeLineWidth,
+    strokeLineCap = StrokeCap.Butt,
+    strokeLineJoin = StrokeJoin.Miter,
+    strokeLineMiter = 1.0f,
+    pathFillType = pathFillType,
+    pathBuilder = pathBuilder
+)
+
+@PublishedApi
```

Comment:
오 이건 처음보는건데 뭔가용

Reply:
- https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-published-api/
- https://kotlinlang.org/docs/inline-functions.html#restrictions-for-public-api-inline-functions

원래는 public inline function에서 non-public API는 참조하지 못하는데요, 왜냐하면 컴파일 할 때 non-public API의 변경사항을 반영하지 못할 수 있기 때문이에요(표현이 좀 애매한 것 같은데 두 번째 링크나 아래 문구 참고해주세요). 

> This imposes certain risks of binary incompatibility caused by changes in the module that declares an inline function in case the calling module is not re-compiled after the change.

아무튼 non-public API 중에서 `internal`로 명시된 API는 `@PublishedApi` 어노테이션을 추가해서 public인 것처럼 컴파일할 때 반영할 수 있다 이렇게 이해하면 될 것 같습니다.
물론 internal이니까 모듈 외부에서는 접근할 수 없는 건 동일하구요

Reply:
이런게 있었군요~!